web/admin/application/languages/es_ES/es_ES.po

```diff
@@ -863,11 +861,22 @@ msgstr "Externa"
 msgid "External Filter destination"
 msgstr "Destino llamada filtro"
 
+msgid "External call"
+msgid_plural "External calls"
+msgstr[0] "Llamada facturable"
+msgstr[1] "Llamadas facturables"
```

Comment:
Is this translation intended? Afaik, not all external calls can be part of an invocie